### PR TITLE
8286199: ProblemList jdk/jshell/ExternalEditorTest.java

### PR DESCRIPTION
**Pull request by RogerRiggs**
Put jdk/jshell/ExternalEditorTest.java on the problem list due to 8286191.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed (1 review required, with at least 1 reviewer)

### Issue
 * [JDK-8286199](https://bugs.openjdk.java.net/browse/JDK-8286199): ProblemList jdk/jshell/ExternalEditorTest.java ⚠️ Issue is not open.

### Reviewers
 * [Daniel D. Daugherty](https://openjdk.java.net/census#dcubed) (@dcubed-ojdk - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/8557/head:pull/8557` \
`$ git checkout pull/8557`

Update a local copy of the PR: \
`$ git checkout pull/8557` \
`$ git pull https://git.openjdk.java.net/jdk pull/8557/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 8557`

View PR using the GUI difftool: \
`$ git pr show -t 8557`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/8557.diff">https://git.openjdk.java.net/jdk/pull/8557.diff</a>

</details>

### CODE REVIEW

File: test/lib-test/ProblemList.txt [hunk 1]
```diff
@@ -37,3 +37,4 @@
 # More than one label is allowed but must be on the same line.
 #
 #############################################################################
+jdk/test/lib/TestMutuallyExclusivePlatformPredicates.java 8286191 generic-all
```
Comment (dcubed-ojdk) [inline test/lib-test/ProblemList.txt:40]: This change shouldn't be here. I think you need to update
your repo to sync with your earlier fix.

**Review by dcubed-ojdk — COMMENTED** [review]
Thumbs up (after resync). This is a trivial fix.

File: test/lib-test/ProblemList.txt (hunk 1 above)
Comment (RogerRiggs) [reply test/lib-test/ProblemList.txt]: Updated, after the merge only the langtools ProblemList.txt is modified

**Review by dcubed-ojdk — APPROVED** [review]
Thumbs up. This is a trivial fix.

**Comment by RogerRiggs** [comment]
ProblemListing the test to clean up the CI.
#8556 seems to have been delayed

**Pull request closed without merging**